src/ldf-client-ui.js

```diff
@@ -5,6 +5,7 @@
 window.jQuery = require('../deps/jquery-2.1.0.js');
 var N3 = require('n3');
 var resolve = require('relative-to-absolute-iri').resolve;
+var markerArray = [];
```

Comment:
Could we do this without global variables. This is considered bad programming practise.

Reply:
I did make it such that it can collect all marker objects. I realised if I make it when I start executing the array it was not being recognized below in the part where I try to use it as I add markers immediately after adding them to the map